lms/static/styles/components/_ErrorDisplay.scss

```diff
@@ -2,13 +2,23 @@
 
 
 .ErrorDisplay {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  overflow-y: scroll;
+  padding-right: 5px;
+  margin-bottom: 10px;
   max-width: 100%;
+  details {
```

Comment:
I literally did not know there was a `<details>` element. TIL! This element is not supported in IE11—we OK with that? Also also: We don't tend to use nested element selectors; perhaps a class could be applied to the `details` element and styled?

Reply:
Sure, I'll add the class and good to know about IE11. I'm gonna just leave this since its LMS and not the sidebar. It has not been an issue so far.